NebulaPatcher/Patches/Dynamic/GameSave_Patch.cs

```diff
@@ -9,12 +9,19 @@ class GameSave_Patch
     {
         [HarmonyPrefix]
         [HarmonyPatch("SaveCurrentGame")]
-        public static void SaveCurrentGame_Prefix(string saveName)
+        public static bool SaveCurrentGame_Prefix(string saveName)
         {
             if (SimulatedWorld.Initialized && LocalPlayer.IsMasterClient)
             {
                 SaveManager.SaveServerData(saveName);
             }
+
+            if (SimulatedWorld.Initialized && !LocalPlayer.IsMasterClient)
+            {
+                return false;
+            }
+
+            return true;
```

Comment:
Should fix also `LoadCurrentGame_Prefix` below I think

Reply:
I am not sure, we do not need to prevent clients to load single-player games when they play in MP, right? Or do we force them to quit to the menu first and then load the new game?